tests/common.rs

```diff
@@ -37,3 +37,28 @@ pub fn setup_pie() -> CairoPie {
 
     runner.get_cairo_pie(&vm).unwrap()
 }
+
+#[allow(unused)]
+pub fn compare_python_output(program: &str, mut vm: VirtualMachine) {
+    let mut rs_output = String::new();
+    vm.write_output(&mut rs_output).unwrap();
+    let rs_output = rs_output.split('\n').filter(|&x| !x.is_empty());
+    let python_output = std::process::Command::new("cairo-run")
+        .arg("--layout=small")
+        .arg(format!("--program={program:}"))
+        .arg("--print_output")
+        .output()
+        .expect("failed to run python vm");
+    let python_output = unsafe { std::str::from_utf8_unchecked(&python_output.stdout) }.to_string();
+    let python_output = python_output
+        .split('\n')
+        .into_iter()
+        .skip_while(|&x| x != "Program output:")
+        .skip(1)
+        .filter(|&x| !x.trim().is_empty())
+        .into_iter();
+    for (i, (rs, py)) in rs_output.zip(python_output).enumerate() {
+        let py = py.to_string().trim().to_string();
+        pretty_assertions::assert_eq!(*rs, py, "Output #{i:} is different");
```

Comment:
please create an issue to improve that

Reply:
nothing needs to be improved it's perfect